PP_APP/client/js/script.js

```diff
@@ -0,0 +1,49 @@
+// Send GET XMLHTTPRequest()
+// const xhr = new XMLHttpRequest();
+// xhr.open('GET', 'http://127.0.0.1:8000/api/display_feed?page=1');
+// xhr.send();
+var display_page_num = 1
+var display_page_count = 1
+
+window.addEventListener("load", function () {
```

Comment:
A couple of comments would be helpful to understand what different parts do. Or at least an overarching docstring.

Reply:
For sure. I'll add comments there. Those variables keep track of the page number as the server will not remember what page it last sent. So we have to keep track of it on the client-side.

Reply:
should we follow the usual practice of Django, adding a template folder within the Django project folder (at the sample level with all the other applications) to contain all the .html files instead of having the client folder?